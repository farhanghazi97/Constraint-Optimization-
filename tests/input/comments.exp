m1:fri 11am
cost:0
