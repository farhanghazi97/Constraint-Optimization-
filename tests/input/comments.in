# Testing comments are ignored
meeting, m1 # ignored
#meeting, m2
   #meeting, m3 #meeting, m4
domain, m1, fri, hard#domain, m1, before 9am, hard
domain, m1, 11am, hard #domain, m1, before 9am, hard
#domain, m1, afternoon, hard
#domain, m2, morning, hard
#domain, m3, after 12pm, hard
#domain, m4, before 6am, hard
