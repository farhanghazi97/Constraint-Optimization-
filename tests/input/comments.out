m1 # ignored:fri 9am
cost:0
